### DISCUSSION TITLE
How to use `FallbackValue` with Image?

### DISCUSSION THREAD

**Discussion by Coloryr** [AvaloniaUI/Avalonia#11428] [Q&A] [answered]
xml
```
<Image
                Name="Image1"
                Width="90"
                Height="90"
                HorizontalAlignment="Left"
                VerticalAlignment="Center"
                Source="{Binding Image^, FallbackValue=LoadImage}" />
```
code
```
 public Task<Bitmap?> Image => GetImage();
    public Bitmap LoadImage => App.LoadIcon;
```
![H5AME5QGIVYEZOIR5Z2G04U](https://github.com/AvaloniaUI/Avalonia/assets/26276037/b0cf040a-368e-432f-b43b-afa68eba25f9)
The loading image is a `GIF`
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/00438b98-6c49-430d-ba47-54759545711c)

**Comment by maxkatz6** [comment]
Binding.FallbackValue has nothing to do with image content. It's used when binding has failed with exception or path property doesn't exist.

**Reply by Coloryr** [reply]
It's my misunderstanding the docs?
![image](https://github.com/AvaloniaUI/Avalonia/assets/26276037/a7982cca-e9bf-4133-b97c-e3c06a05fef4)

**Reply by maxkatz6** [reply]
I just noticed that you use "^" operator on async operation. Then it's fine, as fallback will be used while async operation is working.

We don't suppose git in the Image control (there are some third party controls, and probably Avalonia.Gif too though). 
Does it work with normal non animated image?

**Reply by Coloryr** [reply]
![HWUFCE_T 6S(`Q9E8EY4AO9](https://github.com/AvaloniaUI/Avalonia/assets/26276037/9a3e9aeb-9588-435f-9f81-011a49e00392)

![TX8V$% 63F17K(OFIR(7J@T](https://github.com/AvaloniaUI/Avalonia/assets/26276037/fa96b36c-1df8-4cde-a7c0-b9e72f105fe3)

No work..

**Reply by Coloryr** [reply]
Only display the result image.
Already to use two image to display loading operation.
![$DE)O(YE$)PWM5V`VVSQ55R](https://github.com/AvaloniaUI/Avalonia/assets/26276037/2d59afd0-e943-4c33-8c03-0e2158dbd2a6)

**Reply by maxkatz6** [reply, marked as answer]
I suppose the reason here is that you pass a string to the FallbackValue, but it needs an actual bitmap object here.
You also can try to create a static Bitmap property, and reference it like this:
`Source="{Binding Image^, FallbackValue={x:Static ns:MyStaticClass.FallbackImage}}"`

**Reply by Coloryr** [reply]
> I suppose the reason here is that you pass a string to the FallbackValue, but it needs an actual bitmap object here. You also can try to create a static Bitmap property, and reference it like this: `Source="{Binding Image^, FallbackValue={x:Static ns:MyStaticClass.FallbackImage}}"`

On it work

**Reply by Coloryr** [reply]
![98XEQDQ({8WSE)~}~}_96PC](https://github.com/AvaloniaUI/Avalonia/assets/26276037/95d7bc23-f77b-4ca9-a8da-3f959754d477)
Thanks
